core/modules/notification/components/Notification.ts

```diff
@@ -0,0 +1,17 @@
+import NotificationItem from '../types/NotificationItem'
+
+export const Notification = {
+  computed: {
+    notifications () {
+      return this.$store.getters['notification/notifications']
+    }
+  },
+  methods: {
```

Comment:
maybe add/remove as a names of the functions will be more descriptive (and match Vuex actions)

Reply:
Or spawnNotification (would also fit as a name of Vuex action but addNotification is also descriptive enough, just not so precise)